tests/integration/test_analysisd/test_eps/test_no_eps_configuration.py

```diff
@@ -0,0 +1,137 @@
+import os
+from time import sleep
+import pytest
+
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data, \
+                                              get_simulate_agent_configuration
+from wazuh_testing.modules.eps import event_monitor as evm
+from wazuh_testing.modules.eps import ANALYSISD_STATE_INTERNAL_DEFAULT, PERCENTAGE_PROCESS_MSGS
+
+
+pytestmark = [pytest.mark.server]
```

Comment:
Add tier to the test

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3181/commits/f817116435639fa720611bd68eb90ced7c606062)